config/locales/server.en.yml

```
@@ -104,3 +105,23 @@ en:
   reviewable_score_types:
     needs_review:
       title: "Needs Review"
+
+  user_notifications:
+    chat_summary:
+      deleted_user: "Deleted user"
+      description:
+        one: "You have a new chat message"
+        other: "You have new chat messages"
+      from: "%{site_name}"
+      subject:
+        one: "[%{email_prefix}] New chat message"
+        other: "[%{email_prefix}] New chat messages"
+      unsubscribe: "This email is sent from %{site_link}. Change your %{email_preferences_link}, or %{unsubscribe_link} to unsubscribe."
```

```suggestion
      unsubscribe: "This chat summary is sent from %{site_link} when you are away. Change your %{email_preferences_link}, or %{unsubscribe_link} to unsubscribe."
```

(Better matches equivalent email copy in core.)